gobblin-docs/user-guide/Docker-Integration.md

```diff
@@ -18,68 +18,68 @@ The `gobblin/gobblin-wikipedia` repository contains images that run the Gobblin
 
-The output of the Gobblin-Wikipedia job should now be written to `/home/gobblin/work-dir/job-output`. The `-v` command in Docker uses a feature of Docker called [data volumes](https://docs.docker.com/engine/tutorials/dockervolumes/). The `-v` option mounts a host directory into a container and is of the form `[host-directory]:[container-directory]`. Now any modifications to the host directory can be seen inside the container-directory, and any modifications to the container-directory can be seen inside the host-directory. This is a standard way to ensure data persists even after a Docker container finishes. It's important to note that the `[host-directory]` in the `-v` option can be changed to any directory (on OSX it must be under the `/Users/` directory), but the `[container-directory]` must remain `/home/gobblin/work-dir` (at least for now).
+This example job is correspondent to the [getting started guide](https://gobblin.readthedocs.io/en/latest/Getting-Started/). With the docker image, you can focus on the Gobblin functionalities, avoiding the hassle of building a distribution.
 
-## Gobblin-Standalone Repository
+### Use Gobblin Standalone on Docker for Kafka and HDFS Ingestion 
 
-The Docker images for this repository can be found on Docker Hub [here](https://hub.docker.com/r/gobblin/gobblin-standalone/). These images run a Gobblin standalone service inside a Docker container. The Gobblin standalone service is a long running process that can run Gobblin jobs defined in a `.job` or `.pull` file. The job / pull files are submitted to the standalone service by placing them in a directory on the local filesystem. The standalone service monitors this directory for any new job / pull files and runs them either immediately or on a scheduled basis (more information on how this works can be found [here](Working-with-Job-Configuration-Files#adding-or-changing-job-configuration-files)). Running the Gobblin standalone service inside a Docker container allows Gobblin to pick up job / pull files from a directory on the host filesystem, run the job, and write the output back the host filesystem. All the heavy lifting is done inside a Docker container, the user just needs to worry about defining and submitting job / pull files. The goal is to provide a easy to setup environment for the Gobblin standalone service.
+* To ingest from/to Kafka and HDFS by Gobblin, you need to start services for Zookeeper, Kafka and HDFS along with Gobblin. We use docker [compose](https://docs.docker.com/compose/) with images contributed to docker hub. Firstly, you need to create a [docker-compose.yml](https://github.com/apache/incubator-gobblin/blob/master/gobblin-docker/gobblin-recipes/kafka-hdfs/docker-compose.yml) file(copy the content into your .yml file).
```

Comment:
The link `https://github.com/apache/incubator-gobblin/gobblin-docker/gobblin-recipes/kafka-hdfs/docker-compose.yml` is broken. Did you mean `https://github.com/apache/incubator-gobblin/gobblin-docker/gobblin-recipes/kafka-hdfs/docker-compose.yml`?

Reply:
This is expected, since the file haven't been pushed to master. Please kindly use from my personal fork:
https://github.com/hanghangliu/incubator-gobblin/blob/gobblin-1317-add-docker-recipes-documentations/gobblin-docker/gobblin-recipes/kafka-hdfs/docker-compose.yml

https://github.com/hanghangliu/incubator-gobblin/blob/gobblin-1317-add-docker-recipes-documentations/gobblin-docker/gobblin-recipes/kafka-hdfs/hadoop.env